x/validators/msg.go

```diff
@@ -45,7 +44,7 @@ func (m Pubkey) AsABCI() abci.PubKey {
 
 func (m *SetValidatorsMsg) Validate() error {
 	if len(m.ValidatorUpdates) == 0 {
-		return errors.WithCode(errEmptyValidatorSet, CodeEmptyValidatorSet)
+		return EmptyValidatorErr
```

Comment:
No more info to be passed here, so returning base error is fine. 

Until I add stacktraces, when I will want to do something, but that is another issue...